libs/core/src/lib/shellbar/product-menu/product-menu.component.html

```diff
@@ -20,7 +20,7 @@
         [closeOnOutsideClick]="closeOnOutsideClick"
     >
         <li *ngFor="let item of items" (click)="itemClicked(item, $event)" fd-menu-item>
-            <a [attr.href]="item.link ? item.link : null" fd-menu-interactive>
+            <a [attr.href]="item.link ? item.link : null" [attr.target]="item.newTab ? '_blank' : '_self'" fd-menu-interactive>
```

Comment:
Instead of '_self' you can just pass null so that target attr wont be rendered at all

Reply:
Thank you, i will update